webapp/package.json

```diff
@@ -53,8 +53,8 @@
     "webpack": "webpack/webpack#master"
   },
   "scripts": {
-      "check": "eslint --ext \".jsx\" --ignore-pattern node_modules --quiet .",
-      "build": "webpack",
-      "run": "webpack --progress --watch"
+    "check": "eslint --ext \".jsx\" --ignore-pattern node_modules --quiet .",
```

Comment:
![screen shot 2016-03-23 at 5 34 11 pm](https://cloud.githubusercontent.com/assets/11034289/13985257/8286dd50-f11d-11e5-9fec-70d104ff489a.png)
I think the spacing is inline with the above.